icons/cloth.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4.3 9.7c.4-.4.4-1 0-1.4L2.7 6.7c-.4-.4-.4-1 0-1.4l2.6-2.6c.4-.4 1-.4 1.4 0l1.6 1.6c.4.4 1 .4 1.4 0l1.6-1.6c.4-.4 1-.4 1.4 0l1.6 1.6c.4.4 1 .4 1.4 0l1.6-1.6c.4-.4 1-.4 1.4 0l2.6 2.6c.4.4.4 1 0 1.4l-1.6 1.6c-.4.4-.4 1 0 1.4l1.6 1.6c.4.4.4 1 0 1.4l-1.6 1.6c-.4.4-.4 1 0 1.4l1.6 1.6c.4.4.4 1 0 1.4l-2.6 2.6c-.4.4-1 .4-1.4 0l-1.6-1.6c-.4-.4-1-.4-1.4 0l-1.6 1.6c-.4.4-1 .4-1.4 0l-1.6-1.6c-.4-.4-1-.4-1.4 0l-1.6 1.6c-.4.4-1 .4-1.4 0l-2.6-2.6c-.4-.4-.4-1 0-1.4l1.6-1.6c.4-.4.4-1 0-1.4l-1.6-1.6c-.4-.4-.4-1 0-1.4l1.6-1.6z" />
```

Comment:
These could use arcs instead of cubic Bézier curves, and if we use a radius of √2, we can keep the points on the grid in spite of optimization:
```suggestion
  <path d="M3 5a1.41 1.41 0 0 0 0 2l1 1a1.41 1.41 0 0 1 0 2l-1 1a1.41 1.41 0 0 0 0 2l1 1a1.41 1.41 0 0 1 0 2l-1 1a1.41 1.41 0 0 0 0 2l2 2a1.41 1.41 0 0 0 2 0l1-1a1.41 1.41 0 0 1 2 0l1 1a1.41 1.41 0 0 0 2 0l1-1a1.41 1.41 0 0 1 2 0l1 1a1.41 1.41 0 0 0 2 0l2-2a1.41 1.41 0 0 0 0-2l-1-1a1.41 1.41 0 0 1 0-2l1-1a1.41 1.41 0 0 0 0-2l-1-1a1.41 1.41 0 0 1 0-2l1-1a1.41 1.41 0 0 0 0-2l-2-2a1.41 1.41 0 0 0-2 0l-1 1a1.41 1.41 0 0 1-2 0l-1-1a1.41 1.41 0 0 0-2 0l-1 1a1.41 1.41 0 0 1-2 0L7 3a1.41 1.41 0 0 0-2 0Z" />
```